InertiaCore/Utils/Vite.cs

```diff
@@ -0,0 +1,256 @@
+using System.Text.Encodings.Web;
+using System.Text.Json;
+using Microsoft.AspNetCore.Html;
+using Microsoft.AspNetCore.Mvc.Rendering;
+using System.Text.RegularExpressions;
+
+namespace InertiaCore.Utils;
+
+
+class ViteBuilder
```

Comment:
When we will have the `IViteBuilder` interface, we can make this class `internal`. Because the static methods in the `Vite` class will return the interface. (see more on that below)